governance/pyth_staking_sdk/src/pyth-staking-client.ts

```diff
@@ -710,4 +716,44 @@ export class PythStakingClient {
       undefined,
     );
   }
+
+  public async getTargetAccount(): Promise<TargetAccount> {
+    const targetAccount =
+      await this.stakingProgram.account.targetMetadata.fetch(
+        getTargetAccountAddress(),
+      );
+    return convertBNToBigInt(targetAccount);
+  }
+
+  /**
+   * This returns the current scaling factor between staked tokens and realms voter weight.
+   * The formula is n_staked_tokens = scaling_factor * n_voter_weight
+   */
+  public async getScalingFactor(): Promise<number> {
+    const targetAccount = await this.getTargetAccount();
```

Comment:
Note: this is only valid if someone has cranked the target account (happens whenever someone stakes or unstakes)